code/modules/unit_tests/timer_sanity.dm

```diff
@@ -0,0 +1,3 @@
+/datum/unit_test/timer_sanity/Run()
+	if(SStimer.bucket_count < 0)
+		Fail("SStimer is going into negative bucket count from something")
```

Comment:
@MrStonedOne This may be worth having in SStimer to be checked throughout rounds